docs/source/optimizations/cq/cq_compile_css.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "eb89a144",
```

Comment:
<p>These should be updated to the Markdown syntax for linked images. Not entirely sure if you need to fix what's already being implemented in https://github.com/Infleqtion/client-superstaq/pull/551/. Same for the <code>qss</code> notebook </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>